conf/squid/common/9node-1client-availability-zone.yaml

```diff
@@ -0,0 +1,108 @@
+# Test Suite that deploys 3 AZs in different subnets
+# Deployment for all the ceph daemons , with 9 mon's, 6 mgr's, 9 OSD daemons
```

Comment:
Why do we need 9 mon and 6 mgr daemons ? 

Suggested numbers are - 3 mons and 2 mgrs

Reply:
@pdhiran will we be able to survive with 3 mons and 2 mgrs in 3AZ setup?

Reply:
no. 3 AZ clusters require minimum of 3 mons, 2 Mgr per AZ. Suggested numbers are for regular deployment.